drivers/sensors/pmw3360.c

```diff
@@ -15,6 +15,7 @@ uint16_t pmw33xx_get_cpi(uint8_t sensor) {
     }
 
     uint8_t cpival = pmw33xx_read(sensor, REG_Config1);
+    pmw33xx_write(sensor, REG_Config1, cpival);
```

Comment:
Perhaps add a comment as to why we're writing during a read call?

Reply:
added